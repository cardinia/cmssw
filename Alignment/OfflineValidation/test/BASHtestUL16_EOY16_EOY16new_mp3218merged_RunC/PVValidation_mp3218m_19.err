26-Sep-2019 12:50:31 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/80000/92DEBAFD-C908-E911-8FBD-0CC47A4D7630.root
26-Sep-2019 12:50:35 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/80000/92DEBAFD-C908-E911-8FBD-0CC47A4D7630.root
Begin processing the 1st record. Run 275781, Event 87789, LumiSection 1 on stream 0 at 26-Sep-2019 12:50:43.794 CEST
Begin processing the 1001st record. Run 275781, Event 10348180, LumiSection 8 on stream 0 at 26-Sep-2019 12:54:34.355 CEST
Begin processing the 2001st record. Run 275781, Event 19466275, LumiSection 14 on stream 0 at 26-Sep-2019 12:57:17.560 CEST
Begin processing the 3001st record. Run 275781, Event 28395309, LumiSection 18 on stream 0 at 26-Sep-2019 12:59:53.068 CEST
Begin processing the 4001st record. Run 275781, Event 35059946, LumiSection 22 on stream 0 at 26-Sep-2019 13:03:06.728 CEST
Begin processing the 5001st record. Run 275781, Event 42219152, LumiSection 26 on stream 0 at 26-Sep-2019 13:06:25.383 CEST
Begin processing the 6001st record. Run 275782, Event 1269341099, LumiSection 704 on stream 0 at 26-Sep-2019 13:09:23.990 CEST
Begin processing the 7001st record. Run 275782, Event 1278018203, LumiSection 709 on stream 0 at 26-Sep-2019 13:09:25.417 CEST
Begin processing the 8001st record. Run 275782, Event 1284687993, LumiSection 713 on stream 0 at 26-Sep-2019 13:09:26.839 CEST
Begin processing the 9001st record. Run 275782, Event 1291002997, LumiSection 717 on stream 0 at 26-Sep-2019 13:09:28.444 CEST
Begin processing the 10001st record. Run 275782, Event 1299599272, LumiSection 722 on stream 0 at 26-Sep-2019 13:09:29.905 CEST
Begin processing the 11001st record. Run 275782, Event 1307275284, LumiSection 727 on stream 0 at 26-Sep-2019 13:09:31.533 CEST
Begin processing the 12001st record. Run 275782, Event 1313860262, LumiSection 731 on stream 0 at 26-Sep-2019 13:09:32.819 CEST
Begin processing the 13001st record. Run 275782, Event 1323033252, LumiSection 736 on stream 0 at 26-Sep-2019 13:09:34.508 CEST
Begin processing the 14001st record. Run 275782, Event 1328682206, LumiSection 740 on stream 0 at 26-Sep-2019 13:09:35.927 CEST
Begin processing the 15001st record. Run 275782, Event 1338518504, LumiSection 745 on stream 0 at 26-Sep-2019 13:09:37.396 CEST
Begin processing the 16001st record. Run 275782, Event 1345925414, LumiSection 750 on stream 0 at 26-Sep-2019 13:09:38.577 CEST
Begin processing the 17001st record. Run 275782, Event 1355150349, LumiSection 755 on stream 0 at 26-Sep-2019 13:09:39.885 CEST
Begin processing the 18001st record. Run 275782, Event 1360747237, LumiSection 759 on stream 0 at 26-Sep-2019 13:09:41.442 CEST
Begin processing the 19001st record. Run 275841, Event 2520673, LumiSection 2 on stream 0 at 26-Sep-2019 13:09:43.022 CEST
Begin processing the 20001st record. Run 275841, Event 10857331, LumiSection 8 on stream 0 at 26-Sep-2019 13:09:44.107 CEST
Begin processing the 21001st record. Run 275841, Event 17930053, LumiSection 13 on stream 0 at 26-Sep-2019 13:09:45.376 CEST
Begin processing the 22001st record. Run 275841, Event 24721985, LumiSection 18 on stream 0 at 26-Sep-2019 13:09:46.703 CEST
Begin processing the 23001st record. Run 275841, Event 33015105, LumiSection 24 on stream 0 at 26-Sep-2019 13:09:47.809 CEST
Begin processing the 24001st record. Run 275841, Event 40081351, LumiSection 29 on stream 0 at 26-Sep-2019 13:09:49.141 CEST
Begin processing the 25001st record. Run 275841, Event 48015140, LumiSection 35 on stream 0 at 26-Sep-2019 13:09:50.356 CEST
Begin processing the 26001st record. Run 275841, Event 55749703, LumiSection 40 on stream 0 at 26-Sep-2019 13:09:51.565 CEST
Begin processing the 27001st record. Run 275841, Event 63551762, LumiSection 46 on stream 0 at 26-Sep-2019 13:09:52.786 CEST
Begin processing the 28001st record. Run 276097, Event 964136623, LumiSection 506 on stream 0 at 26-Sep-2019 13:09:54.066 CEST
26-Sep-2019 13:09:54 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/80000/92DEBAFD-C908-E911-8FBD-0CC47A4D7630.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[8.349MB/8.349MB][100%][==================================================][8.349MB/s]  [8.349MB/8.349MB][100%][==================================================][8.349MB/s]  
