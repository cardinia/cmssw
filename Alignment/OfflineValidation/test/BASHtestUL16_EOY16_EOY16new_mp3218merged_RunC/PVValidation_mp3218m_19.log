000 (3189548.019.000) 09/26 11:22:57 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189548.019.000) 09/26 12:49:58 Started transferring input files
	Transferring to host: <188.184.212.133:9618?addrs=188.184.212.133-9618+[2001-1458-301-10c--100-7d]-9618&noUDP&sock=5123_5de9_3944>
...
040 (3189548.019.000) 09/26 12:49:58 Finished transferring input files
...
001 (3189548.019.000) 09/26 12:49:58 Job executing on host: <188.184.212.133:9618?addrs=188.184.212.133-9618+[2001-1458-301-10c--100-7d]-9618&noUDP&sock=3277_53f1_3>
...
006 (3189548.019.000) 09/26 12:50:07 Image size of job updated: 24132
	11  -  MemoryUsage of job (MB)
	10780  -  ResidentSetSize of job (KB)
...
006 (3189548.019.000) 09/26 12:55:08 Image size of job updated: 562008
	544  -  MemoryUsage of job (MB)
	556476  -  ResidentSetSize of job (KB)
...
006 (3189548.019.000) 09/26 13:00:08 Image size of job updated: 564356
	545  -  MemoryUsage of job (MB)
	557292  -  ResidentSetSize of job (KB)
...
040 (3189548.019.000) 09/26 13:10:17 Started transferring output files
...
040 (3189548.019.000) 09/26 13:10:17 Finished transferring output files
...
005 (3189548.019.000) 09/26 13:10:17 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:18:35, Sys 0 00:00:07  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:18:35, Sys 0 00:00:07  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6312  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	6312  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     8594        1   2462774 
	   Memory (MB)          :      545     2000      2000 
...
