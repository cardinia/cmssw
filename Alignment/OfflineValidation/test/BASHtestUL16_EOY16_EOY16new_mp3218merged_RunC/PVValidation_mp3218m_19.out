Job started at  Thu Sep 26 12:49:59 CEST 2019
batch dir: /pool/condor/dir_15751 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_275781_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 8.5M -rw-r--r--. 1 tkello zh 8.4M Sep 26 13:10 PVValidation_mp3218m_275781.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:49 PVValidation_mp3218m_275781_cfg.py -rw-r--r--. 1 tkello zh 5.0K Sep 26 13:10 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:50 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:49 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:50 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:49 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:49 var
Job ended at  Thu Sep 26 13:10:17 CEST 2019
